packages/common/src/ui/components/navigation/ExternalNavLink.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
```

Comment:
all this to have a nav link that looks the same as the others, but doesn't link to a relative path, or both with selected state

Reply:
good point! thanks